inginious/frontend/pages/tasks.py

```diff
@@ -342,7 +342,7 @@ def submission_to_json(self, task, data, debug, reloading=False, replace=False,
             tojson["title"] = _("An internal error occurred. Please retry later. "
                                 "If the error persists, send an email to the course administrator.")
 
-        tojson["title"] += " " + _("[Submission #{submissionid}]").format(submissionid=data["_id"])
+        tojson["title"] += " " + _("[Submission #{submissionid} from the <b>{submissionDate}</b>]").format(submissionid=data["_id"], submissionDate=data["submitted_on"].replace(microsecond=0))
```

Comment:
I'd use format rather than replacing some value of the date.
See https://docs.python.org/fr/3/library/datetime.html#datetime.date.strftime